### DISCUSSION TITLE
Windows portable version: omit unnecessary files from zip archive(s)

### DISCUSSION THREAD

**Discussion by lintalist** [Qalculate/qalculate-gtk#487] [Ideas]
I've only just found qalculate and using the portable version. 

It seems to me there are a lot of unnecessary files included in the Windows zip version(s). I think the `share\icons\Adwaita\` folder can be omitted (it doesn't seem to affect the program when I delete it). There are 5800+ image files in that folder alone which is a lot of data 31mb for "nothing".

If that if it is indeed the case there are perhaps other folders that simply can be omitted, this will make the download(s) smaller, updating (unpacking) faster and consume a lot less disk space.

**Comment by Stefanoko** [comment]
I strongly second lintalist's suggestion. Thousands of small files waste precious space on USB-sticks with their large cluster sizes. It also creates overhead of managing the data, sync and copy and virus scans take a lot longer.

After deleting the Adwaita folder from the portable Windows version, the physical space occupied by Qalculate went down from 1GB to 220 MB.

**Comment by hanna-kn** [comment]
If all icons are removed from the folder, a few button icons will be missing, arrows will change appearance, and many icons will be missing from file dialogs. It does however seem as if only the SVG icons (scalable folders) are needed.

**Comment by Stefanoko** [comment]
Thank you, Hanna-kn. I will experiment with the svg-icons to see what I need and what to leave out.
If you know of other files and folders not absolutely necessary, please let us know.
I appreciate it.

**Comment by lintalist** [comment]
Cheers - removing PNGs would already be a big change!
But even withing the SVGs I doubt everything is needed (angel emoji, games etc etc) so I suspect more could be removed.
